src/gui/Window.qml

```diff
@@ -100,4 +111,9 @@ ApplicationWindow {
 
         Behavior on scale { HNumberAnimation { overshoot: 3; factor: 1.2 } }
     }
+
+    HTrayIcon {
```

Comment:
The tray icon isn't a reusable component, it is specific to the Window, so it should be in the *gui* folder instead of *gui/Base*. The `H` prefix is used for component names that can conflict with standard ones (e.g. `Button` → `HButton`), so you can just name it `TrayIcon` in this case.